github/helpers/githubSDK.ts

```diff
@@ -631,14 +647,12 @@ export async function getPullRequestComments(
 export async function getIssuesComments(
     http: IHttp,
     repoName: string,
-    access_token: string,
     issueNumber: string | number
 ) {
     const response = await http.get(
         `https://api.github.com/repos/${repoName}/issues/${issueNumber}/comments`,
         {
             headers: {
-                Authorization: `token ${access_token}`,
                 "Content-Type": "application/json",
```

Comment:
This could be handled by a `if-else` block. Similar to this pseudo code below. Try mimicking this in setting up your logic. Similar has been done in a lot  of the existing GitHub SDK functions written by us. 

```
function exampleFunction(token?: string, url: str){
   if(token){
        http_request_with_token(token,url);
    }else{
        http_request_without_token(token,url);
    }
}

```